Documentation/biblio/geom.bib

```diff
@@ -151890,6 +151907,26 @@ @book{botsch2010polygon
   author={Botsch, Mario and Kobbelt, Leif and Pauly, Mark and Alliez, Pierre and L{\'e}vy, Bruno and others},
   publisher =   "A K Peters, Ltd.",
   year={2010}
+=======
+@inproceedings{desbrun1999implicit,
+  title={Implicit fairing of irregular meshes using diffusion and curvature flow},
+  author={Desbrun, Mathieu and Meyer, Mark and Schr{\"o}der, Peter and Barr, Alan H},
+  booktitle={Proceedings of the 26th annual conference on Computer graphics and interactive techniques},
+  pages={317--324},
+  year={1999},
+  organization={ACM Press/Addison-Wesley Publishing Co.}
+}
+
+@article{amenta2001power,
+  title={The power crust, unions of balls, and the medial axis transform},
+  author={Amenta, Nina and Choi, Sunghee and Kolluri, Ravi Krishna},
+  journal={Computational Geometry},
+  volume={19},
+  number={2},
+  pages={127--153},
+  year={2001},
+  publisher={Elsevier}
+>>>>>>> gsoc13-CurveSkeleton-new_package-xgao-old
```

Comment:
See the merge-conflict markers? I wonder how it can work. I thought that the last testsuite had tested the branch entirely!